src/main/java/core/basesyntax/Loterry.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Loterry {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        return new Ball(colorSupplier.getRandomColor(),random.nextInt(1,100));
```

Comment:
```suggestion
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(1, 100));
```

Reply:
Check code style and save numbers to constant

Reply:
not fixed